spec/controllers/subclassed_pages_controller_spec.rb

```diff
@@ -1,34 +1,34 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe SubclassedPagesController do
   render_views
 
-  describe 'on GET to /subclassed_pages/also_exists' do
-    before { get :show, :id => 'also_exists' }
+  describe "on GET to /subclassed_pages/also_exists" do
+    before { get :show, id: "also_exists" }
```

Comment:
In a previous PR for testing against the beta I added a shim for handling this -

https://github.com/thoughtbot/high_voltage/blob/master/spec/support/http_method_shim.rb

Which was borrowed from Clearance.